docs/howto/how-to-build-runtime.md

```diff
@@ -8,11 +8,12 @@ If you are going to build this project, the following modules must be installed
 
 - CMake
 - Boost C++ libraries
+- Gnome core C libraries
 
 In the Ubuntu, you can easily install it with the following command.
 
 ```
-$ sudo apt-get install cmake libboost-all-dev
```

Comment:
The following is beyond this PR. It is about our policy of default build, not the content of this PR (document):

First of all, personally I prefer to minimize the dependency as much as possible.
`libglib2.0-dev` is required only for `npud`, which is under-developing and furthermore, `onert` does not require `npud`. It would be better to remove this dependency for most users who just want to build runtime as the guide says (`how-to-build-runtime`).

Reply:
I fully understood what you mean and I agreed with you. 
Maybe we need more opinions and a consensus about how the build system must be. 
 
In my opinion, we can add an environment variable that decides 'build npud or not'.